### PR TITLE
Login - Signup

### PR DESCRIPTION
**Pull request by eliawk**

### CODE REVIEW

File: src/scripts/actions/UserActions.js [hunk 1]
```diff
@@ -117,9 +117,9 @@ export const setupKeystore = () => async (
 
   // Case 2: we have a secured wallet is localStorage
   if (walletStringSecure && ACTIVATE_SECURE_WALLET) {
-    console.log('Try to open encrypted keystore')
+    // console.log('Try to open encrypted keystore')
     // Need to ask the PIN
-    dispatch(openModal(MODAL.ASK_PASSWORD))
+    // dispatch(openModal(MODAL.ASK_PASSWORD))
```
Comment (bent0b0x) [inline src/scripts/actions/UserActions.js]: remove commented lines?

File: test/functional-tests/app.spec.js [hunk 2]
```diff
@@ -7,12 +7,12 @@ describe('Portal application', () => {
...
-    })
-  })
+  // describe('Nav bar', () => {
+  //   it("should render the user's PTI balance", () => {
+  //     browser.url('http://localhost:8080')
+  //     browser.waitUntil(() =>
+  //       browser.getText('[data-test-id="nav-pti-balance"]').includes('PTI')
+  //     )
+  //   })
+  // })
```
Comment (bent0b0x) [inline test/functional-tests/app.spec.js]: Do we need to fix these tests?

File: test/functional-tests/profile.spec.js [hunk 3]
```diff
@@ -459,94 +418,94 @@ describe('Profile and accounts workflow:', function () {
...
+  //     browser.click('#logoutBtn')
+  //     browser.waitForClickable('#nav-profile')
+  //     assertUserIsNotLoggedIn(browser)
+  //
+  //     login(browser, 'foobar')
+  //
+  //     waitForUserIsLoggedIn(browser)
+  //     assertUserIsLoggedIn(browser)
+  //   })
+  // })
```
Comment (bent0b0x) [inline test/functional-tests/profile.spec.js]: Are these just temporarily commented out?

Comment (eliawk) [reply test/functional-tests/profile.spec.js]: we are not using so maybe we could remove it but it's a temporary reminder

File: test/functional-tests/app.spec.js (hunk 2 above)
Comment (eliawk) [reply test/functional-tests/app.spec.js]: now when you arrive to the app there is no more the `PTI balance`, so this test don't make sense

File: src/scripts/actions/UserActions.js (hunk 1 above)
Comment (eliawk) [reply src/scripts/actions/UserActions.js]: it's temporary, because now i've removed the `askPassword` at the beginning but maybe it will come back|

Comment (bent0b0x) [reply src/scripts/actions/UserActions.js]: I would just delete it, we can easily add it back in later

File: test/functional-tests/app.spec.js (hunk 2 above)
Comment (bent0b0x) [reply test/functional-tests/app.spec.js]: Can we delete them then?

File: test/functional-tests/profile.spec.js (hunk 3 above)
Comment (bent0b0x) [reply test/functional-tests/profile.spec.js]: If we are _sure_ we are going to need this later I'd just add a `.skip`, otherwise delete

**Pull request merged**